libraries/helpers.rb

```diff
@@ -103,5 +103,3 @@ def ubuntu_18?
     end
   end
 end
-
-Chef::DSL::Universal.include(Nginx::Cookbook::Helpers)
```

Comment:
Why is this being removed?

Reply:
I'm including it in the resource `action_class` rather than polluting the whole DSL